misc/chip_data.json

```diff
@@ -192,7 +192,7 @@
         "5858a52e-54fb-4bb2-80a0-ec5fce88b3ef": {
           "Name": "AI Has Line of Sight To Target",
           "ReadonlyTypeParams": {
-            "T": "(AI | Combatant | CreationObject | PatrolPoint | Player | Vector3)"
```

Comment:
Same as other review, would be good to change `CustomObject` back to `CreationObject` for now.

Reply:
Done!